docs/source/examples/examples.myst

```diff
@@ -4,12 +4,26 @@
 ---
 maxdepth: 2
 ---
-ibmq-backends.rst
-hamiltonians.rst
-simple_landscape.rst
+ibmq-backends.myst
+hamiltonians.md
+simple_landscape.myst
 maxcut-demo.myst
 pyquil_demo.myst
 cdr_api.md
 mitiq-paper/mitiq-paper-codeblocks
 pec-tutorial.myst
 ```
+
+## Links to the examples for My Binder
```

Comment:
Should "My Binder" be just one word? Or just "Binder"?

Reply:
The convention specified by the service is mybinder.org, will update to this.